master/getting-started/kubernetes/installation/gce.md

```diff
@@ -2,49 +2,21 @@
 title: Deploying Calico and Kubernetes on GCE
 ---
 
-These instructions allow you to set up a Kubernetes cluster with Calico networking on GCE using the [Calico CNI plugin][cni-plugin]. This guide does not setup TLS between Kubernetes components or on the Kubernetes API.
+There are several possibilities for [deploying Kubernetes on GCE][running-GCE-on-K8s]
+and with minimal adjustments Calico can be [installed](.) too.
```

Comment:
Yeah not really necessary, I just thought it was nice to provide a link to one of the many ways to install k8s. I can drop it if you don't think it is useful.